libs/domain/cart/add/src/add.component.ts

```diff
@@ -58,22 +68,29 @@ export class CartAddComponent extends ProductMixin(
     </oryx-button>`;
   }
 
-  protected min = computed(() => {
-    const value = 1;
-    if (this.$product()) {
-      this.shadowRoot
-        ?.querySelector<QuantityInputComponent>('oryx-cart-quantity-input')
-        ?.reset?.();
-    }
-    return value;
+  @elementEffect()
+  protected reset = effect(() => {
+    if (this.$product()) this.input?.reset?.();
+  });
```

Comment:
```suggestion
  protected reset = () => {
    if (this.$product()) this.input?.reset?.();
  };
```

Reply:
We need to reset the input whenever the SKU is changing